middleware/http/wasm/benchmark_test.go

```diff
@@ -0,0 +1,135 @@
+package wasm
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+	"testing"
+
+	"github.com/dapr/kit/logger"
+
+	"github.com/dapr/components-contrib/internal/httputils"
+	"github.com/dapr/components-contrib/metadata"
+	"github.com/dapr/components-contrib/middleware"
+	"github.com/dapr/components-contrib/middleware/http/wasm/basic"
+)
+
+const parallel = 10
+
+func Benchmark_native(b *testing.B) {
+	benchmarkAll(b, func(next http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			if httputils.RequestURI(r) == "/v1.0/hi?name=panda" {
+				httputils.SetRequestURI(r, "/v1.0/hello?name=teddy")
+			}
+			next.ServeHTTP(w, r)
+		})
+	})
+}
+
+func Benchmark_basic(b *testing.B) {
+	md := metadata.Base{Properties: map[string]string{
+		"path":     "./basic/example/example.wasm",
+		"poolSize": strconv.Itoa(parallel),
+	}}
+
+	l := logger.NewLogger(b.Name())
+	l.SetOutput(io.Discard)
```

Comment:
fyi benchmarks don't do any logging, and I checked there's no perf difference after using dapr/kit logging